aws/cloudformation/s3_backed_subdomain.yml

```diff
@@ -42,6 +42,16 @@ Resources:
     Properties:
       AccessControl: PublicRead
       BucketName: !Ref BucketName
+      CorsConfiguration:
+        CorsRules:
+          - AllowedMethods: [GET]
+            AllowedOrigins:
+              [
+                '*.code.org',
```

Comment:
I can't tell from the [AllowedOrigin docs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ManageCorsUsing.html#cors-allowed-origin) if I should separate this origin into two origins (one for HTTP and one for HTTPS). If anyone knows for sure, let me know! I can also test this with a fallback implemented once I deploy an instance of a component that consumes this resource.

Reply:
I think this is right. Your `*` should cover it both protocols. These cloudformation docs have some examples. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-s3-bucket.html#cfn-s3-bucket-corsconfiguration

Reply:
Can we use CloudFormation string interpolation to make the base domain name configurable using the value of the template Parameter `Domain`? Something like this (switching from bracket array to hyphen array syntax to make the string substitution easier). I think it's safe to assume local development environments listen on port 3000.:
```
            AllowedOrigins:
              - !Sub "*.${Domain}"
              # Permit local host development environments
              - *.code.org:3000
              - http://localhost-studio.code.org:*
```

Reply:
yep! updated the origins and applied the changeset to make sure everything still looks good on localhost